calvin/calvin.py

```diff
@@ -173,7 +173,7 @@ def add_ag_region_sinks(self):
       links.upper_bound = maxub
       links['link'] = links.i.map(str) + '_' + links.j.map(str) + '_' + links.k.map(str)
       links.set_index('link', inplace=True)
-      self.df = self.df.append(links.drop_duplicates())
+      self.df = self.df._append(links.drop_duplicates())
```

Comment:
Mustafa, overall these seem like good fixes, but this reference to `_append` makes me a little uneasy. Probably fine for this week, but worth looking for something that doesn't rely on an underscore method in the long run because it may change behaviors without warning.